src/components/editor/InputListLOC.jsx

```diff
@@ -130,3 +138,4 @@ const mapDispatchtoProps = dispatch => ({
 })
 
 export default connect(mapStatetoProps, mapDispatchtoProps)(InputListLOC)
+export { InputListLOC }
```

Comment:
I haven't seen this much in our components. What does this do?

Reply:
Opps, that was an experiment I forgot to remove. It was supposed to export the "undecorated" version if the component (i.e. the unwrapped component). I'm not sure if it actually did or not, but I discarded trying to use that.